extension/popup/popupController.jsx

```diff
@@ -96,7 +132,8 @@ export const PopupController = function() {
     addListeners();
     updateExamples();
     updateLastIntent();
-    startRecorder();
+
+    if (skipRecording === false) startRecorder();
```

Comment:
Hrm... I can understand the logic here, if the timer is running then don't start the recorder. But this means the whole tool will be disabled until the timer is done, right? The user doesn't have to say anything, so it feels like we should still have the recorder running even when there's a timer.